fullstack-mpp/build.gradle.kts

```
@@ -0,0 +1,83 @@
+import org.jetbrains.kotlin.gradle.targets.js.webpack.*
+
+plugins {
+    id("org.jetbrains.kotlin.multiplatform") version "2.1.0"
+}
+
+kotlin {
+    targets {
+        js("frontend", IR) {
+            browser {
+                testTask {
+                    // TODO: disable browser tests since we can"t run it on teamcity agents yet
+                    enabled = false
+                }
+                webpackTask {
+                    mainOutputFileName = "output.js"
+                }
+                binaries.executable()
+            }
+        }
+        jvm("backend") {
+        }
+    }
+
+    sourceSets.forEach {
+        it.dependencies {
+            implementation(project.dependencies.enforcedPlatform("io.ktor:ktor-bom:3.0.2"))
+        }
+    }
+
+    sourceSets {
+        commonTest {
+            dependencies {
+                implementation("org.jetbrains.kotlin:kotlin-test-annotations-common")
+                implementation("org.jetbrains.kotlin:kotlin-test-common")
+            }
+        }
+    }
+
+    sourceSets {
+        val backendMain by getting {
+            dependencies {
+                implementation("io.ktor:ktor-server-netty")
+                implementation("io.ktor:ktor-server-html-builder")
+                implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+                implementation("ch.qos.logback:logback-classic:1.5.12")
+            }
+        }
+        val backendTest by getting {
+            dependencies {
+                implementation("org.jetbrains.kotlin:kotlin-test")
+                implementation("org.jetbrains.kotlin:kotlin-test-junit")
+            }
+        }
+        val frontendMain by getting {
+            dependencies {
+                implementation("org.jetbrains.kotlin:kotlin-stdlib-js")
+                implementation("org.jetbrains.kotlin:kotlin-test-js")
+                implementation("io.ktor:ktor-client-core")
+                implementation("io.ktor:ktor-client-js")
+                // specify ws version explicitly to fix webpack task
+                implementation(npm("ws", "8.18.0")) // TODO should be removed, after ktor 3.0.3 release (KTOR-7912)
+            }
+        }
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+val backendJar = tasks.getByName<Jar>("backendJar") {
```

This case is slightly different. Here we don't want to configure this task instantly, so it's better to use lazy API returning `TaskProvider` instead of `Task`. The task will be configured only when we call `.get()` on the provider or execute this task.

```suggestion
val backendJar = tasks.named<Jar>("backendJar") {
```

---

You are right!
I could build the project successfuly just with this change.